lib/glific/registrations/registration.ex

```diff
@@ -27,7 +27,8 @@ defmodule Glific.Registrations.Registration do
     :has_confirmed,
     :ip_address,
     :terms_agreed,
-    :support_staff_account
+    :support_staff_account,
+    :notion_page_id
```

Comment:
will it be `page id` or `row id` in a page?

Reply:
In Notion a row is actually a page and we use pages api to create the row. https://developers.notion.com/docs/working-with-databases